fuse/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "GoodSubGraphsFuse",
+  "name": "good-subgraphs-fuse",
```

Comment:
this whole commit should be reverted

Reply:
this goes against npm package name naming convention